### ISSUE TITLE
Lack of documentation for FST file and for material JSON

### ISSUE THREAD

**Issue by ksuprynowicz** [vircadia/vircadia-docs-sphinx#178]
There's no document concerning what parameters can be used inside an avatar FST file.
Documentation for material JSON lacks some information, for example there's no information about variable name used for emissive texture URL.

**Comment by JulianGro** [comment]
Regarding FST documentation: https://github.com/vircadia/vircadia-docs-sphinx/pull/167

**Comment by SilverfishVR** [comment]
Yes FST are a bit mysteries, I think many new users don't realize it that it is just a text file with optional overrides/additions for models, or even that it can be used for normal models, not just avatars.
```emissiveMap``` is in the API docs but it can be confusing that there is both:
https://apidocs.vircadia.dev/Entities.html#.Material
And:
https://apidocs.vircadia.dev/Graphics.html#.Material
but they are not quite the same, almost though
